CONTRIBUTING.md

```diff
@@ -1,18 +1,126 @@
-## Adding quests
+Thanks for your interest in contributing to StreetComplete! 👍 There are many easy tasks you can do, even without programming knowledge, to create a better app for all users of StreetComplete and drive the OpenStreetMap project forward.
 
-[Do you have an idea for a new quest? Read this!](https://github.com/westnordost/StreetComplete/wiki/Adding-new-Quests-to-StreetComplete) (_before_ opening an issue)
+Even if you do not find something to do in this list, using StreetComplete, testing it and giving constructive feedback is always a valuable contribution too.
 
-You want to contribute a quest, but don't have any idea, which? Look [at the existing issues](
-https://github.com/westnordost/StreetComplete/issues?q=is%3Aissue+is%3Aopen+label%3A%22new+quest%22+sort%3Areactions-%2B1-desc). Sorted by :+1: reactions you can also see which are the most requested quests.
+Content:
+* [Translating the app](#translating-the-app)
+* [Helping mappers by solving their notes](#helping-mappers-by-solving-their-notes)
+* [Joining discussions](#joining-discussions)
+* [Testing and reporting issues](#testing-and-reporting-issues)
+   * [Issues of dependencies](#issues-of-dependencies)
+   * [Suggesting new quests](#suggesting-new-quests)
+* [Improving documentation](#improving-documentation)
+* [Development](#development)
+   * [Developing new quests](#developing-new-quests)
 
-## Translations
+## Translating the app
 
-Translations are managed at POEditor. Please contribute them there. Manual translations as Pull Requests will not be merged and do not help the project.
+- ☑️ requires no programming knowledge
+- ☑️ requires no OpenStreetMap knowledge
```

Comment:
* These checkboxes are weird.
* I think the texts in the bullet points do not make that much sense. Whether or not one requires knowledge of one or the other kind should be apparent in the paragraph just below. If not, I dare to say that it is not clearly written.
* Also, it's confusing that they are negated: _"⬜️ requires no OpenStreetMap knowledge"_ -> "Requires not no OSM knowledge", so err...
* This specific instance is even wrong. For some translations, you actually do need to know some knowledge about OSM. Some translations need to be as precise as the tags are.

Reply:
see [my comment before](https://github.com/westnordost/StreetComplete/pull/1005#issuecomment-379574179) on why I do it, but I agree it kinda looks weird.

I could also strike-through not checked boxes, but maybe it really still does not help. If the text there requires you to think (evaluate boolean state :wink:) it's indeed not a good doc.

So I drop that and get back to a solution with short bold statements.
As explained before, I like that solution, because users can skim over the doc and find something without programming knowledge e.g., if they are not a programmer. Of course, that's obvious for some parts, but is not for some others. Not everyone wants to read the whole doc.